lib/plaid/institution.rb

```diff
@@ -77,9 +77,116 @@ def self.all
     #
     # Returns an Institution instance or raises Plaid::NotFoundError if
     # institution with given id is not found.
-    def self.get(id)
+    def self.get!(id)
```

Comment:
`!` also means "throws exceptions instead of returning nil". But I decided to ditch this. Most of the gem methods throw exceptions in case of error, we'd need to mark them all with `!`.